### DISCUSSION TITLE
Conservation Status

### DISCUSSION THREAD

**Discussion by tomwhite101** [mcguirepr89/BirdNET-Pi#436] [Ideas]
Hi, 
It would be great if you could pull in a conservation status with the record in order to be able to sort/notify on that tag.

i.e. I would like to be notified immediately if an amber or red list species is observed.

I know you are flat out with development anyway, but I just thought to drop this here for the future.
I note that eBird has the classification with the record of each species so possibly a source to pull from?

Very best,
Tom

**Comment by Phaeton** [comment]
I like this idea a lot. I find myself always checking the conservation status on Wikipedia whenever a new to my bird is tagged.

**Comment by Phaeton** [comment]
It looks like IUCN has an API and fairly favorable license although IANAL.

 [IUCN API Docs](https://apiv3.iucnredlist.org/api/v3/docs)

**Reply by ehpersonal38** [reply]
> You may not repost, or redistribute to any third party, IUCN Red List Data in whole, or in part, by any means, including (but not limited to) electronic formats such as internet postings, web downloads, through web services including utilization of **The IUCN Red List APIs**, through interactive web maps that grant users download access, KML files or through file transfer protocols, electronic mailing, faxing, archiving in a public data, redistributing via a computer network, digital storage, memory stick or other electronic media or device, except as may otherwise be expressly permitted by IUCN in writing through a formal Waiver. See section [15](https://www.iucnredlist.org/terms/terms-of-use#15.%20Definitions%20of%20Terms) for Definitions of Terms).

Does this mean we can't pass the API data to Gmail or Telegram notifications since they're third parties?

Honestly their terms seem pretty unfavorable for an organization trying to raise awareness, but 'tis what it is.

**Reply by Phaeton** [reply]
Ugh, I must have been wearing my rosy colored glasses yesterday. I’ll have to do some more digging to see if something else is out there.

**Reply by ehpersonal38** [reply]
https://pif.birdconservancy.org/avian-conservation-assessment-database/

This seems like a good resource. It shows the proper way to cite it, so I'd say we're good to go in using it for non-commercial use.
